routes/buzz-words.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const saveBuzzWords = require('../save-buzz-words');
+const express = require('express');
+const router = express.Router();
+const bodyParser = require('body-parser');
+const urlEncoded = bodyParser.urlencoded({ extended: false });
+
+// get the buzzwords from POST requests, store them as their own objects.
+// have an array of objects (buzzwords) that you can send when the client requests with GET.
+
+// the file directory in this file is relative to /buzzwords
+// the path '/buzzwords' is set to '/' here because of app.use('/buzzwords', buzzwords);
+
+router.get('/', (req, res) => {
+  res.send(saveBuzzWords);
+});
+
+// POST request helper:
+// validates that an object with the same buzzword does not already exist in the array.
+router.foundMatch = (checkObj) => {
```

Comment:
You shouldn't modify router this way. If you need a helper function, you should just declare it as a regular function without putting it on router.